spec/controllers/report_controller_spec.rb

```diff
@@ -1223,6 +1223,33 @@
       EvmSpecHelper.local_miq_server
     end
 
+    context "when generating reports" do
+      render_views
+      let!(:rpt) { MiqReport.first }
```

Comment:
I don't think you don't need the `!` here.

Reply:
Good point, without it I can use FactoryGirl as well. For some reason it was failing with it on local machine :confused: